src/DotLiquid.Tests/BlockTests.cs

```diff
@@ -78,7 +78,7 @@ public void TestWithCustomTagFactory()
             Template.RegisterTagFactory(new CustomTagFactory());
             Template result = null;
             Assert.DoesNotThrow(() => result = Template.Parse("{% custom %}"));
-            Assert.AreEqual("I am a custom tag"+Environment.NewLine, result.Render());
+            Assert.That(result.Render(), Is.EqualTo("I am a custom tag" +Environment.NewLine));
```

Comment:
```suggestion
            Assert.That(result.Render(), Is.EqualTo("I am a custom tag" + Environment.NewLine));
```

Reply:
I let VS reformat somewhat and am committing just space changes.